### PR TITLE
hacks.mozilla article

### PR DESCRIPTION
**Pull request by benparsons**

### CODE REVIEW

File: hacks/hacks.mozilla.org.1.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Dweb: Matrix
+
+## Author info: Full name, photo of yourself, and 1-2 sentence bio
+
+## What is Matrix?
+
+Matrix is an open standard for *interoperable*, *decentralised*, *real-time* communication over IP. It provides a standard HTTP API for publishing and subscribing to real-time data in specified channels, which means it can be used to power Instant Messaging, VoIP/WebRTC signalling, Internet of Things communication, and anything else that could be transmitted over HTTP. The most common use of Matrix today is as an Instant Messaging platform.
```
Comment (ara4n) [inline hacks/hacks.mozilla.org.1.md:7]: "and anything else that can be expressed as JSON and needs to be transmitted in realtime over HTTP" perhaps?

(also, it'd be good if these were wordwrapped to aid review, but not the end of the world)

File: hacks/hacks.mozilla.org.1.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+* Matrix is *interoperable* in that it can freely communicate with other platforms. Matrix messages are JSON, and easy to parse. Bridges are provided to enable communication with other platforms.
```
Comment (ara4n) [inline hacks/hacks.mozilla.org.1.md:9]: interoperable in that it follows an [open standard](https://matrix.org/docs/spec) and can freely...

File: hacks/hacks.mozilla.org.1.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+* Matrix is *decentralised* - there is no central server. To communicate on Matrix, you connect your client to a single "homeserver" - this server then communicates with other homeservers. For every room you are in, your homeserver will maintain a copy of the history of that room. This means that no one homeserver is the host or owner of a room if there is more than one homeserver connected to it.
+
+## Why create another messaging platform?
+
+The initial goal is to fix the problem of fragmented IP communications: letting users message and call each other without having to care what app the other user is on - making it as easy as sending an email.
+
+In future, we want to see Matrix used as a generic HTTP messaging and data synchronisation system for the whole web, enabling IoT applications through a single unified, understandable interface.
```
Comment (ara4n) [inline hacks/hacks.mozilla.org.1.md:16]: "IoT and other applications" perhaps

**Comment by ara4n** [comment]
lgtm other than nits - thanks :)

File: hacks/hacks.mozilla.org.1.md (hunk 1 above)
Comment (benparsons) [reply hacks/hacks.mozilla.org.1.md:7]: oops. yeah I've been messing with VS Code settings, I guess this works for me but doesn't play nicely outside

**Pull request closed without merging**